README.md

```diff
@@ -14,17 +14,31 @@
 [![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
 [![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=security_rating)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
 
-## Overview
+# Overview
 
-Crossplane is an open source multicloud control plane. It introduces workload and resource abstractions on-top of existing managed services that enables a high degree of workload portability across cloud providers. A single crossplane enables the provisioning and full-lifecycle management of services and infrastructure across a wide range of providers, offerings, vendors, regions, and clusters. Crossplane offers a universal API for cloud computing, a workload scheduler, and a set of smart controllers that can automate work across clouds.
+Crossplane is an open source multicloud control plane to manage your
+cloud-native applications and infrastructure across environments, clusters,
+regions and clouds. It enables provisioning and full-lifecycle management
+ of applications and managed services from your choice of cloud using `kubectl`.
 
-<h4 align="center"><img src="docs/media/arch.png" alt="Crossplane" height="400"></h4>
+Crossplane can be installed into an existing Kubernetes cluster to add managed
+service provisioning or deployed as a dedicated control plane for multi-cluster
```

Comment:
I think we could split this up, I had a hard time tracking the modifiers ("or [it can be] deployed").  

Perhaps, "Once deployed, Crossplane can act as a dedicated..."

Reply:
Yes let’s do another pass at word-smithing the docs. Would prefer to get this landed and do a follow on PR for that if possible.